.github/workflows/release.yml

```diff
@@ -138,11 +138,11 @@ jobs:
             image: ghcr.io/cross-rs/arm-unknown-linux-gnueabihf:main
             runs-on: buildjet-4vcpu-ubuntu-2204
           - target: arm-unknown-linux-musleabihf
-            platform: linux_armv6l
+            platform: muslinux_armv6l
```

Comment:
Can we add the second `l` here? `musllinux` pls

Reply:
(Because python likes the double `l`: https://packaging.python.org/en/latest/specifications/platform-compatibility-tags/#musllinux)